cluster-autoscaler/estimator/estimator.go

```diff
@@ -26,18 +26,14 @@ import (
 )
 
 const (
-	//BasicEstimatorName is the name of basic estimator.
-	BasicEstimatorName = "basic"
 	// BinpackingEstimatorName is the name of binpacking estimator.
 	BinpackingEstimatorName = "binpacking"
+	// BinpackingOldEstimatorName is the name of the older binpacking estimator.
+	OldBinpackingEstimatorName = "oldbinpacking"
```

Comment:
shouldn't this one be deprecated to and marked as to be removed soon?

Reply:
Its in the desc of OldBinpackingEstimator. Just like it was in the case of BasicEstimator